vars/pythonSetupPyPipeline.groovy

```diff
@@ -24,23 +24,35 @@ def call(Map pipelineParams) {
     pipelineParams["dockerFilename"] = "Dockerfile"
   }
   if (!pipelineParams.dockerBuildArgs) {
-    pipelineParams["dockerBuildArgs"] = ""
+    pipelineParams["dockerBuildArgs"] = "--no-cache --network host"
```

Comment:
are those params mandatory? We should consider having base and additional params. Base can be overridden if needed(can they?), additional are appended to the base ones.

Reply:
Also are those parameters required in any Jenkins instance (specifically the --network host part) or is it just our instance?

Reply:
Some are specially for our setup, some are just base ones like the no-cache to ensure we built always from scratch the layers to avoid using something cached which might not be available anymore (eg. packages from ubuntu mirrors).

I agree that those settings due to our setup should not be here but I think they will soon disappear once I switch to the new setup. So therefore I would propose to keep them until then